docs/index.md

```diff
@@ -0,0 +1,120 @@
+# Index
+
+  * About - About this project
+  * Installation - Installing Islandora CLAW using Ansible
+  * Software - Islandora CLAW dependencies and their versions
+  * Vagrant - virtual machine information
+  * Usage - Using CLAW
+  * Contributing - How to contribute
+  * Documentation - How to build this documentation in mkdocs
+
+___
+
+## About CLAW Vagrant Islandora 7.x-2.x
+
+[Islandora](http://islandora.ca) is an open-source software framework designed to help institutions and organizations and their audiences collaboratively manage, and discover digital assets using a best-practices framework.  Islandora was originally developed by the University of Prince Edward Island's Robertson Library, but is now implemented and contributed to by an ever-growing international community.
+
+Islandora CLAW consists of:
+
+  * [Sync](camel/sync/README.md) - Event driven middleware based on Apache Camel that synchronizes a Fedora 4 JCR with a Drupal CMS.
+  * [Islandora](drupal/islandora/README.md) - Fedora 4 Repository module
+  * [Install](install/README.md) - The is a development environment virtual machine for the Islandora CLAW project. It should work on any operating system that supports Ansible, Git, VirtualBox and Vagrant.
+
+---
+
+## Installation 
+
+The [installation section](install/README.md) provides and overview on how to create a virtual development environment.
+
+## Using Ansible 
+
+* List of commands here
+
+## Ansible Playbook structure
```

Comment:
I think this section goes a little too detailed. I think the directory structure of the repo changes fairly quickly, and including it here assures that it will be outdated soon.